examples/contrib/import_har/readhar.py

```diff
@@ -100,6 +104,9 @@ def request_to_flow(self, request_json: dict) -> http.HTTPFlow:
 
         new_flow.client_conn.timestamp_start = timestamp_start
         new_flow.client_conn.timestamp_end = timestamp_end
+
+        new_flow.request.http_version = http_version_req
```

Comment:
This needs testing. If I look into the HAR files, I see cases where we have `httpVersion` set to `""`, or to `http/2.0`. We typically have something like `HTTP/2` in there though!

Also this should update the test results so that tests pass. :)